docs/content/docs/custom-resource/reference.md

```diff
@@ -182,6 +182,7 @@ This serves as a full reference for FlinkDeployment and FlinkSessionJob custom r
 | upgradeMode | org.apache.flink.kubernetes.operator.api.spec.UpgradeMode | Upgrade mode of the Flink job. |
 | allowNonRestoredState | java.lang.Boolean | Allow checkpoint state that cannot be mapped to any job vertex in tasks. |
 | savepointRedeployNonce | java.lang.Long | Nonce used to trigger a full redeployment of the job from the savepoint path specified in initialSavepointPath. In order to trigger redeployment, change the number to a different non-null value. Rollback is not possible after redeployment. |
+| autoscalerResetNonce | java.lang.Long | Nonce used to reset the autoscaler metrics, parallelism overrides and history for the job. This can be used to quickly go back to the initial user-provided parallelism settings without having to toggle the autoscaler on and off. In order to trigger the reset behaviour simply change the nonce to a new non-null value. |
```

Comment:
nits:

> This can be used to quickly go back to the initial user-provided parallelism settings without having to toggle the autoscaler on and off.

This sentence is comparing the new and old approach. For existing users, they understand this approach to reset parallelism. For new autoscaler users, this sentence may be not needed.

Or if you think it's needed, would you mind moving it to the end of the paragraph?

> In order to trigger the reset behaviour simply change the nonce to a new non-null value.

How to update it to `In order to trigger the reset behaviour simply, change the nonce or number to a different non-null value.`? It's will unify the description of `savepointRedeployNonce` and `autoscalerResetNonce`.